assets/screens/shared_components.onj

```diff
@@ -4,11 +4,39 @@ use Common;
 import "imports/colors.onj" as color;
 import "config/game_config.onj" as gameConfig;
 
+var worldWidth = 1600.0;
+var worldHeight = 900.0;
+
 navBarUsedAssets: [
     "red_wing_cm", "location_sign_salem", "statusbar_background", "nav_bar_background", "location_name_aqua_dulche",
-    "location_name_salem", "statusbar_option", "reserves_texture", "statusbar_background_left",
+    "location_name_salem", "statusbar_option", "reserves_texture", "statusbar_background_left"
+],
+
+backpackAssets: [
+    "backpack_1",
+    "backpack_2",
+    "backpack_3",
+    "backpack_4",
+    "backpack_5",
+    "backpack_1_hover",
+    "backpack_2_hover",
+    "backpack_3_hover",
+    "backpack_4_hover",
+    "backpack_5_hover",
+    "backpack_deck_background",
+    "backpack_backpack_background", //This name intended, not an error
```

Comment:
because it is only the background for the backpack (right) half, the left half is deck. Therefore, since my personal naming convention for assets is "type_info", and type=backpack  and info=backpack_background, since there are two similar but not equal backgrounds